src/abi.h

```diff
@@ -45,6 +45,12 @@ typedef struct dhcp_overrides {
     guint metric;
 } NetplanDHCPOverrides;
 
+typedef struct ipv6_ra_overrides {
+    gboolean use_dns;
```

Comment:
`use-dns` should probably be a `NetplanTristate` instead of a boolean and initialized with `NETPLAN_TRISTATE_UNSET`. It will make things a bit simpler in the code that checks if it's enabled or not.